### ISSUE TITLE
Motion Alarms and Tamper alarms stay on and do not go off

### ISSUE THREAD

**Issue by jeroenvollenbrock** [athombv/homey#2267]
This is a combined issue of multiple related device-specific issues regarding alarms that turn on and do not get turned off.

**What did you try to achieve?**
Make an motion or tamper alarm turn off.

**What did you expect as result?**
The alarm turns off after the condition becomes false.

**What was the result?**
The alarm state stays on.

**Comment by robiebab** [comment]
don't forget this one: https://github.com/athombv/homey-issues-dp/issues/59 could this be the beginning of this issue?

**Comment by CyberSponk** [comment]
Not Resolved for Philio-Device PST02-A in 1.5.7-Exp.

**Comment by WeeJeWel** [comment]
I have fixed this issue and it will be shipped in Homey v1.5.8.

**Comment by corvdniet** [comment]
Updated to 1.5.8 (experimental). Shortly after the update of Homey the (Fibaro) motion kept on. After a couple of minutes it went off. If that is the way it works now, I can live with it.

**Comment by pakacom** [comment]
I also update to 1.5.8 experimental.
no fix for the philips zigbee motion sensors unfortunately
they motion stay on again after a while and do not go off
log code : 08E4A60862

**Comment by jghaanstra** [comment]
Confirmed it's not fixed with 1.5.8 experimental. Updated Homey and my Fibaro sensor was still having the tamper alarm constanly activated. I tampered with the device to trigger the actual tamper alarm and it was cancelled. But some minutes later (approx. 10) out of the blue the tamper alarm went off again and is not cancelled anymore.

**Comment by CyberSponk** [comment]
**Not** fixed in 1.5.8-exp. Updated ~10 hours ago. WakeUp-Time of Devices 6 hours. Tamper-Alarm is On.

**Comment by CyberSponk** [comment]
You forgott these one: https://github.com/Inversion-NL/com.philio/issues/36

**Comment by Inversion-NL** [comment]
Just to confirm: my Fibaro Smoke sensor (1 out of 5) still reports tamper alarm.

**Comment by corvdniet** [comment]
Did a couple of restarts to find out the current behaviour ...
No motion or tamper after restart. When I triggered motion, the sensor reports motion but stays on. Only after triggering tamper the motion and tamper go out. After this "ritual" the sensor seems to work "normal", but only time will tell

**Comment by philsniff** [comment]
seems ok now. i will look at it ecrey day ;-)

**Comment by jsm198** [comment]
I’m with 1.5.8. Same issue as corvdniet. After reboot the motion trigger stays on until I triggered the tampering sensor, then motion went off.

**Comment by corvdniet** [comment]
I'm on 1.5.11-rc.1 (Experimental) now. My Fibaro Motion Sensor works as expected ... about 5 minutes after (re)start off Homey. Did a couple of restarts to be sure. Hope this behaviour stays stable.

**Comment by selzilla** [comment]
exactly the same problems with my homey.
bug still exists in firmware 1.5.11-rc.1

**Comment by CyberSponk** [comment]
Not resolved in Homey 1.5.11-EXP for me. Tamper-Alarm always on.

**Comment by 2benl** [comment]
Same here with Tamper alarm from a Fibaro movement sensor

**Comment by selzilla** [comment]
When will this problem finally be solved?

**Comment by WeeJeWel** [comment]
We are working on an update of the Fibaro app. If that doesn't solve the issue, we will dive in deeper.

**Comment by CyberSponk** [comment]
Got Philio-Devices > No Problem last Year when i got Homey in 02/17. So it seems to be an issue by Homey itself and not by the app. But i'm not sure.

**Comment by WeeJeWel** [comment]
Most Z-Wave devices share the same code in their app, which we have updated. So it might be a problem across all apps.

**Comment by Veldkornet** [comment]
I don't know what all was changed in Fibaro app v1.5.18 - 18.05.18....
But this problem still exists in this version for me anyway.

Although, as per https://github.com/athombv/com.fibaro/issues/185 it's not all modules. Only my v2 Door Window sensors have the problem. The v1 sensors work fine.

**Comment by canedje** [comment]
I do have the same problem with Fibaro motionsensor as mentioned above (Motion alarm and burglar alarm stays on)
I think it is only Fibaro, because all my Neo PIRS do not have this problem.

**Comment by cbho** [comment]
After the latest update to 1.5.18, one of my Fibaro motion detectors began to work as supposed to regarding cancellation of tamper alarm - it is cancelled as supposed to.
However, the other didn't.
Some time ago, trying to get rid of the on-going tamper alarm, I changed "Tamper sensitivity" to 0 (disable) to see if that would solve the problem, which it didn't:
Tamper alarm was still on in Homey, although no tamper alert was sent from the device. Since the tamper alarm wasn't triggered by the device, a cancellation message wasn't sent either - and Homey of course then keep the alarm on.
There have been communication about tampering alarms are turned on when Homey reboot and not cancelled until devices have been manipulated manually.
After changing the setting on my device to send tamper alarms - and therefore also cancel them - the behavior of the sensor is normalised.
Could part of trouble be, that the state of tamper alarms (at least for Fibaro devices) are by default set to "on" in Homey after rebooting?

**Comment by jghaanstra** [comment]
1.5.18 seems to have fixed my issue. The tamper alarm is cancelled and there are no "out of the blue" alarms anymore.

**Comment by gvdijnsen** [comment]
I stumbled upon this thread while looking for solutions for the erratic behaviour of my Wave+ Fibaro Motion Sensor. I seem to have similar symptoms, i.e. tamper alarm staying on for extended periods of time, most of the time the motion detection does not register with Homey. It used to work flawless for several days after inclusion and then the trouble started. I am on version 1.5.18, but the problems persist. Is there anything else I should do like exclude/include the device or reset it for the fix to work?

**Comment by selzilla** [comment]
I can confirm that. There is often no motion detected by the motion sensors.
the fibaro app is no longer usable. I'll soon be ready to sell homey. even my old vera can do it better

**Comment by BBEHannes** [comment]
Maybe a dumb idea but I have the feeling that I only have these “stale status” of my fibaro z-wave motion sensor whenever it meshes through one of my other (eotec) router devices. As soon as it connects directly to homey the motion works as expected. 

Could this help?

**Comment by EdwardDumay** [comment]
I also confirm that the Fibaro app does not work properly. No motion detection, motion detection stays on like the sabotage alarm.
According to the support of Athom  a new version is in a test phase right now.

**Comment by vstankov-zz** [comment]
I have registered two Fibaro Window/Door Sensor 2 and still getting Tamper Alarm active constantly.
Homey v. 1.5.11
Fibaro app v.1.5.18

**Comment by EdwardDumay** [comment]
I had the same issue yesterday with a multi sensor of Fibaro. I restarted the Fibaro app. That fixed it.

**Comment by vstankov-zz** [comment]
I've updated the Fibaro app to beta: v.2.1.0, added new Door/Window Sensor 2 adn still getting the tamper alarm constantly on:
![image](https://user-images.githubusercontent.com/7319002/46944781-a3a73800-d07b-11e8-97a5-0cdf89f37f86.png)

I was able to add the Flood sensor successfully - before it had problems with the tamper alarm constantly on, too.

Best Regards,
Vladimir

**Comment by hive** [comment]
I have the same issue with the Ecobee Apps sensors, affecting all 8 sensors, always show 'motion detected', Is this related or should I open a new issue?

**Comment by pandersson85** [comment]
So, anybody working on this ? Nothing has happened for months and I'm about to toss this crap out the window. My patience is about to run out and I'm about to go to Fibaro Homecenter instead.

**Comment by EdwardDumay** [comment]
Hi all,


I think there is not much attention paid to all the Z-wave supported
devices and Z-wave in general. There is much effort in getting more devices
to be supported by Homey and not in the quality.
I also looked at Fibaro home centre 2. Nice device. I also read there will
be a succesor. If the succesor is released and I still have problems at
that time I will switch to it.

But I still have hope that it will work at the end. Maybe complaining at
the support department of Athom can help.







Op 16 nov. 2018 8:36 p.m. schreef "pandersson85" <notifications@github.com>:

So, anybody working on this ? Nothing has happened for months and I'm about
to toss this crap out the window. My patience is about to run out and I'm
about to go to Fibaro Homecenter instead.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/athombv/homey/issues/2267#issuecomment-439503340>, or mute
the thread

**Comment by Veldkornet** [comment]
I didn't see that this was fixed anywhere, but I just wanted to confirm that I still have the problem on my "Fibaro Door/Window Sensor 2" after updating to Fibaro app v2.1.8 - 22.11.18 on Homey 1.5.12.

I only updated, didn't re-pair devices.

**Comment by YDind** [comment]
On my side, I also noticed that beside the Homey z-wave range issue (that I
solved by adding a powered device 1 meter away), the Fibaro door/window
sensors have weak tamper switch.

I took me some time to « customize » them by adding some tape to ensure the
micro switch is well closed when the device is installed on the window
frame.



Le ven. 23 nov. 2018 à 13:53, Veldkornet <notifications@github.com> a
écrit :

**Comment by Veldkornet** [comment]
Actually, since updating to 2.1.8 I’ve noticed that while while the workaround will work initially (manually triggering the tamper alarm); a while later the alarm comes back on. 

So actually it’s even worse now than it was before.

**Comment by PatrickvanD** [comment]
Same problem with my Hue Motion sensors.  Only work-arround is to remove the device from Homey and add them as a new device. This works only for a few hours/days. 

Alter the update to 1.5.13 two days ago it got worse. Also have problems adding the device. 

Please Athom, do something about it.

**Comment by alpol017** [comment]
Homey 1.5.13 rc latest beta.
Fibaro app 2.1.8 beta
I have restored my fibaro motion sensors Z-wave to factory default.
I have re-add them to homey and haven’t changed their default settings.
And problem still occurs!
Sometimes homey doesn’t get “yes” motion. So my light wouldn’t be turned on.
Sometimes it stays yes forever. So I had to make flows, to turn lights off. Otherwise light would be on forever!
Problem started for me in February 2018, after I update to the latest fibaro app version. Before everything was grate!
10 months later problem still there!
I had to replace some fibaro sensors with neo coolcam, and they are working great on homey.
Dear athom! Please, make this issue the most important!

**Comment by Lokaltog** [comment]
This is most likely not a problem with the fibaro sensors. I've been having these issues with homey and all my fibaro sensors since I first got homey. I haven't updated the sensor firmware (I don't have the fibaro controller) but I've been trying all the latest homey versions. I've really tried getting this to work, pairing and unpairing, moving devices around, resetting the network, you name it. I've been following this thread and hoping that they'd be able to fix this issue but my patience has run out. I ended up throwing out homey and purchasing a raspberry pi with z-wave/zigbee USB dongles and installing home assistant on it.

All my sensor issues have miraculously completely disappeared with home assistant and my entire setup is far more responsive. It was also quite a bit cheaper than Homey. It's a bit more work getting things working, but if you're a power user just don't bother with homey, go for home assistant or something else right away. No more false alarms or stuck sensors, the ~0.5s lag when controlling lights has completely disappeared, and I no longer have the issue with some zigbee lights randomly not responding.

I wish I could get the nice looking homey hardware with everything contained in one device with home assistant running on it.

I'm betting the homey guys could make a fortune limiting themselves to making just the hardware and shipping the devices with home assistant running on them, or at least opening the devices up for running custom firmware. I'd be surprised if the rumored homey version 2.0 firmware will even come close to the possibilities that are currently available for home assistant.
